src/classes/BGE_BatchGiftEntry_UTIL.cls

```diff
@@ -163,10 +160,48 @@ public with sharing class BGE_BatchGiftEntry_UTIL {
         return fieldApiNames;
     }
 
+    private static GE_Template.Template deserializeTemplate(DataImportBatch__c batch) {
+        GE_Template.Template template = (GE_Template.Template) JSON.deserialize(
+                batch.Form_Template__r.Template_JSON__c,
+                GE_Template.Template.class
+        );
+        return template;
+    }
+
+    private static String getNewBatchGiftEntryVersionFields() {
+        return DataImport__c.Additional_Object_JSON__c.getDescribe().getName();
+    }
+
+    private static Boolean isNewBatchGiftEntryVersion(DataImportBatch__c batch) {
+        return batch.Batch_Gift_Entry_Version__c >= 2.0;
+    }
+
+    private static DataImportBatch__c getDataImportBatch(Id batchId) {
+        DataImportBatch__c batch = [
+                SELECT Batch_Gift_Entry_Version__c,
+                        Form_Template__r.Template_JSON__c
+                FROM DataImportBatch__c
+                WHERE Id = :batchId
+        ];
+        return batch;
+    }
+
+    @TestVisible
+    private static List<String> getDataImportElevateFields() {
+        List<String> elevateFieldApiNames = new List<String>();
+        if (GE_PaymentServices.getInstance().isElevateCustomer()) {
+            elevateFieldApiNames = new List<String>{
+                DataImport__c.Payment_Declined_Reason__c.getDescribe().getName()
```

Comment:
This can be `String.valueOf(DataImport__c.Payment_Declined_Reason__c)`.  No need to get the whole describe.

Reply:
updated!